src/index.js

```diff
@@ -12,8 +12,8 @@ const VueShowdown = {
     }
 
     // set default options (override flavor)
-    for (const [key, value] of Object.entries(options)) {
-      showdown.setOption(key, value)
+    for (const key in this.options) {
```

Comment:
It should be `options`, not `this.options`.

Reply:
Right !